subprojects.md

```diff
@@ -0,0 +1,184 @@
+# Jupyter Subprojects
+
+Project Jupyter is composed by many subprojects, each focusing on different aspects of the Jupyter ecosystem. 
+Regarding their governance, some subprojects have their own Council and SSC (Software Steering Committee), and the others are governed directly by the Project SSC.
+
+The major subprojects with their own formal Subproject Council are:
```

Comment:
I would recommend removing this and combining it with the section that you have below Official Subprojects with SSC representation. It's a bit confusing as it implies that some of these such as xeus kernel has their own representation instead of being part of Kernels. I do like the descriptions that you added. I think consolidating into one section would be less confusing. :sunny:

Reply:
I would also encourage you to review the governance documents to ensure that the way the Project Jupyter is described here is the same as that. https://jupyter.org/governance/list_of_subprojects.html If you want to make changes or improvements requesting feedback from the EC would be valuable. Essentially, we want to avoid telling different stories in different places. For the consumer of this information I think it can be confusing that we have the Jupyter story in multiple places. There are governance docs, there is jupyter.org, and there is Jupyter meta documentation.

Reply:
Thank you very much! I'm working out this content to integrate/merge with the already existing. I'll make that clear.